src/test/resources/reportportal.properties

```diff
@@ -0,0 +1,4 @@
+rp.endpoint=http://10.11.211.101:8080
```

Comment:
This is out configuration, which obviously only works behind out firewall. For your Travis-CI runs, you'll need to provide the correct properties for this environment.